NonWeb.md

```diff
@@ -20,8 +21,19 @@ both browsers and node.js. This situation is expected to be first encountered
 with POSIX features such as file I/O. In that respect, WebAssembly would err
 towards standardizing existing practice through libraries, and let developers
 choose which libraries to use.
+Similarly, the WebGL and SDL interface (
+[exposed by emscripten](http://kripken.github.io/emscripten-site/docs/porting/multimedia_and_graphics/OpenGL-support.html)
+in C/C++) could be made into a shared spec with non-browser mobile/desktop
+environments, allowing for cross platform browser & non-browser games and other
+things.
 
 The WebAssembly spec will not try to define any large portable libc-like
 library. However, certain features that are core to WebAssembly semantics that
 are found in native libc *would* be part of the core WebAssembly spec as either
 primitive opcodes or a special builtin module (e.g., `sbrk`, `mmap`).
+
+In general, by keeping the non-browser path such that it doesn't require too
+much "browser emulation" (JS/DOM dependencies), WebAssembly could be used as
+a universal binary standard for many platforms, with huge benefits in
+portability, tooling and language-agnosticity (this being the first widely
+supported "bytecode" that supports C/C++ level semantics).
```

Comment:
> "browser emulation" (JS/DOM dependencies)

I'd like to keep this section specific and not handwave about "browsers" and some need for "emulation". I proposed #251, and it make sense to mention the DOM specifically as an API that NonWeb use cases need not provide, or other specific things people might be concerned about.

I'm also not really comfortable with the rest of the language in this paragraph. "Universal", "binary standard", "first", "widely supported" and "bytecode" are all phrases which may lead to widely varying interpretations. It seems sufficient to me to add mobile and desktop (and IoT, rock on, @jfbastien!) to the use cases at the top.